src/cosmic_ray/commands/init.py

```diff
@@ -2,43 +2,54 @@
 import logging
 from typing import Iterable
 import uuid
-
 from cosmic_ray.ast import get_ast, ast_nodes
 import cosmic_ray.modules
-from cosmic_ray.work_item import MutationSpec, ResolvedMutationSpec, WorkItem
+from cosmic_ray.work_item import ResolvedMutationSpec, WorkItem
 from cosmic_ray.plugins import get_operator
 from cosmic_ray.work_db import WorkDB
 
 log = logging.getLogger()
 
 
-def _all_work_items(module_paths, operator_names) -> Iterable[WorkItem]:
+def _all_work_items(module_paths, operator_cfgs) -> Iterable[WorkItem]:
     "Iterable of all WorkItems for the given inputs."
+
     for module_path in module_paths:
         module_ast = get_ast(module_path)
 
-        for op_name in operator_names:
-            operator = get_operator(op_name)()
+        for operator_cfg in operator_cfgs:
+            operator_name = operator_cfg["name"]
+            if "args" not in operator_cfg:
```

Comment:
I think this conditional would be better expressed as `operator_cfg.get('args', [{}])`

Reply:
that is, it would be clearer to readers.